src/couch/src/couch_httpd_auth.erl

```diff
@@ -695,3 +707,12 @@ authentication_warning(#httpd{mochi_req = Req}, User) ->
         "~p: Authentication failed for user ~s from ~s",
         [?MODULE, User, Peer]
     ).
+
+-spec get_config_hash_algorithms() -> list(atom()).
+get_config_hash_algorithms() ->
+    HashAlgorithmStr = string:to_lower(
```

Comment:
it's not necessary to force to lower case. we only care that the items match supported hash algorithm names.

Reply:
removed